contracts/test/MockSpokePool.sol

```diff
@@ -37,9 +37,12 @@ contract MockSpokePool is SpokePoolInterface, SpokePool {
         _setDepositQuoteTimeBuffer(buffer);
     }
 
-    function initializeRelayerRefund(bytes32 relayerRepaymentDistributionProof) public override {
-        _initializeRelayerRefund(relayerRepaymentDistributionProof);
+    function initializeRelayerRefund(bytes32 relayerRepaymentDistributionRoot, bytes32 slowRelayFulfillmentRoot)
+        public
+        override
+    {
+        _initializeRelayerRefund(relayerRepaymentDistributionRoot, slowRelayFulfillmentRoot);
```

Comment:
I know you've changed this through the play but do we think that changing these variable names would make our linting a bit easier? `relayerRepaymentDistributionRoot` is a _very_ long variable name. would `relayerRepaymentRoot` be better? and `slowRelayFulfillmentRoot` to `slowRelayRoot`? My goal is that we have a lot of functions and variables that are wrapping onto multiple lines and would perhaps not wrap if they were a bit shorter.

dont worry about this for now but just something to think about. IMHO as much as posible having code on _one_ line will make things a lot easier to read and generally makes the code look much cleaner.

Reply:
Yeah, I'll leave for a follow-up. We've had a lot of back and forth on naming in this PR, and just to avoid doing a bunch more find-and-replace, we can do this in one shot in a follow up if that's cool.